app/lib/screens/login_screen.dart

```diff
@@ -211,6 +214,35 @@ class LoginScreenState extends State<LoginScreen> {
         ]));
   }
 
+  Widget _buildAdvancedSettings(BuildContext context) {
+    return Card(
+        child: Padding(
+            padding: const EdgeInsets.all(8),
+            child: Column(children: [
+              Row(mainAxisAlignment: MainAxisAlignment.start, children: [
+                Checkbox(
+                  value: _forceAndroidDownloadButton,
```

Comment:
I think it might look better having the checkbox beneath the URL field.  
Also the URL field might be more important / frequently used.

Reply:
Done